### PR TITLE
Plots

### PR DESCRIPTION
**Pull request by mumichae**
Add basic plots for MAPS output

### CODE REVIEW

File: rules/evaluation.smk [hunk 1]
```diff
@@ -2,50 +2,103 @@
 Evaluate different 3' UTR subsets with MAPS score
 Includes preparation of interval formats, gnomAD hail table, local & CGP runs
 """
+from snakemake.remote.GS import RemoteProvider as GSRemoteProvider
```
Comment (pdiakumis) [inline rules/evaluation.smk:5]: Nice, forgot that Snakemake can handle remote files.

File: scripts/maps_gcp.py [hunk 2]
```diff
@@ -26,7 +26,7 @@
 
     parser = argparse.ArgumentParser(description='Compute MAPS score')
     parser.add_argument(
-        '-o', '--output', required=True, help='Directory for MAPS hail table'
+        '-o', '--output', required=True, help='Csv file for pandas dataframe with MAPS'
```
Comment (pdiakumis) [inline scripts/maps_gcp.py]:
```suggestion
        '-o', '--output', required=True, help='CSV file for pandas dataframe with MAPS'
```

File: scripts/plots.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
+"""
+Plots comparing all variant subsets
+"""
+# flake8: noqa
+
+import pandas as pd
+from plotnine import *  # pylint: disable=W0614,W0401
+
+df = pd.read_csv(snakemake.input[0], sep='\t')
```
Comment (pdiakumis) [inline scripts/plots.py:11]: Maybe comment what sort of input is coming in here.

File: scripts/plots.py [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
```
Comment (pdiakumis) [inline scripts/plots.py:12]: Lots of love for plotnine, thanks for discovering it :-)

**Review by pdiakumis — APPROVED** [review]
Looks great Michaela! I only had very minor comments so you can merge as-is.
I've spotted a typo, some word inconsistencies and re-worded one part in the README in a separate PR (I couldn't add comments about them in this PR since those chunks weren't modified here).

One of the things I like to do with Snakemake rules is include a one-line comment of what its main functionality is, no matter if the rule name is self-explanatory or not. Just a habit. I find it helps when the Snakefile becomes too long and the rules become more and more complex. 

I love the way you're linking scripts in Snakemake. I never tried doing that (can even use Rmd files, who knew!) and wish I had done so earlier.
Also great to know about plotnine.

File: scripts/plots.py (hunk 3 above)
Comment (mumichae) [reply scripts/plots.py:11]: Added a comment in the docstring at the beginning of the file

**Comment by mumichae** [comment]
Thanks @pdiakumis for the comments and fixes. I included your patch commit here. Some of my comments on MAPS output are still fairly general, mainly because things are still WIP and subject to change.

**Comment by pdiakumis** [comment]
Thanks for the changes, looks awesome!

**Pull request merged**